nautobot/extras/utils.py

```diff
@@ -514,6 +514,10 @@ def refresh_job_model_from_job_class(job_model_class, job_class, job_queue_class
 
     try:
         with transaction.atomic():
+            default_job_queue, _ = job_queue_class.objects.get_or_create(
+                name=job_class.task_queues[0] if job_class.task_queues else settings.CELERY_TASK_DEFAULT_QUEUE,
+                defaults={"queue_type": JobQueueTypeChoices.TYPE_CELERY},
```

Comment:
Thinking toward the future, at some point we might want a `NAUTOBOT_DEFAULT_QUEUE_TYPE` config variable for all of these auto-creation cases. Not blocking at this time.